### PR TITLE
Pp 7688 connector selfservice pipeline

### PR DESCRIPTION
**Pull request by oswaldquek**
Connector and selfservice pipeline for staging and prod.

### CODE REVIEW

**Comment by oswaldquek** [comment]
Build currently fails for production deploys with
```
Error: Error creating IAM Role production-2-selfservice-FARGATE-iam_for_ecs_exec_ssm: AccessDenied: User: arn:aws:sts::092359438320:assumed-role/concourse/terraform-prod-assume-role is not authorized to perform: iam:CreateRole on resource: arn:aws:iam::092359438320:role/production-2-selfservice-FARGATE-iam_for_ecs_exec_ssm
	status code: 403, request id: 5babcea8-47c7-412a-9653-362ed2c23883
```

File: ci/pipelines/deploy-to-production.yml [hunk 1]
```diff
@@ -95,6 +95,20 @@ resources:
       repository: govukpay/cardid
       variant: release
       <<: *aws_prod_config
+  - name: connector-ecr-registry-prod
+    type: registry-image-resource-1-1-0
+    icon: docker
+    source:
+      repository: govukpay/cardid
```
Comment (mrlumbu) [inline ci/pipelines/deploy-to-production.yml:102]:
```suggestion
      repository: govukpay/connector
```

File: ci/pipelines/deploy-to-production.yml [hunk 2]
```diff
@@ -95,6 +95,20 @@ resources:
...
+      variant: release
+      <<: *aws_prod_config
+  - name: selfservice-ecr-registry-prod
+    type: registry-image-resource-1-1-0
+    icon: docker
+    source:
+      repository: govukpay/cardid
```
Comment (mrlumbu) [inline ci/pipelines/deploy-to-production.yml:109]:
```suggestion
      repository: govukpay/selfservice
```

**Review by mrlumbu — CHANGES_REQUESTED** [review]
Couple of typos

**Review by mrlumbu — APPROVED** [review]

**Pull request merged**